### PR TITLE
[CS] Code Style round 2 fixes for libraries/src

### PR DESCRIPTION
**Pull request by photodude**
Pull Request for Issue code style fixes

### Summary of Changes
- Please consider an empty line before the if statement
- No blank line found after control structure
- Expected 0 spaces after opening bracket; 1 found
- Expected 1 space before "?"; 0 found
- Expected 1 space after ".="; 2 found

[Automatically fixed with Joomla code standards 2.0.0 PHPCS2-alpha2 fixers](https://github.com/joomla/coding-standards/releases/tag/2.0.0-alpha2)

None of the manual only fixes have been applied

### Testing Instructions
Merge by code review

### Expected result
code style has been applied as listed above, old code style testing on drone does not error.

### Actual result
code style had not been applied. [Autofixers from the Joomla code standards 2.0.0 PHPCS2 alpha2](https://github.com/joomla/coding-standards/releases/tag/2.0.0-alpha2) were used to implement fixable code style

### Documentation Changes Required
none

### CODE REVIEW

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 75da35b5d6f87de073368d43e93861ee68211c0c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19178">issues.joomla.org/tracker/joomla-cms/19178</a>.</sub>

File: libraries/src/Table/Nested.php [hunk 1]
```diff
@@ -313,6 +313,7 @@ public function moveByReference($referenceId, $position = 'after', $pk = null, $
 		{
 			echo "\nMoving ReferenceId:$referenceId, Position:$position, PK:$pk";
 		}
+
 		// @codeCoverageIgnoreEnd
```
Comment (wilsonge) [inline libraries/src/Table/Nested.php]: For the code coverage ignore statements is this some kind of crazy special case where we actively don't want an extra new line?

Comment (mbabker) [reply libraries/src/Table/Nested.php]: Actually, I'd rather just strip the code coverage annotations out of the code.  They just artificially indicate code that shouldn't be counted in code coverage metrics for whatever reason, they're only useful if you're aiming for trying to publish a 100% coverage report.

Comment (photodude) [reply libraries/src/Table/Nested.php]: Make sense to me. Let's strip the code coverage annotations out of the code.

**Comment by zero-24** [comment]
Merged by review

**Pull request merged**